DGN/Views/Articles/Edit.cshtml

```diff
@@ -4,55 +4,63 @@
-                </div>
-                <div class="form-group">
-                    <label>Upload Article Image</label>
-                    <input type="file" name="ImageFile" id="ImageFile" class="form-control-file">
-                </div>
-                <div class="form-group">
-                    <input type="submit" value="Save" class="btn btn-primary" />
+<div class="container">
+    <section id="formSec" class="">
+        <div class="container d-flex align-items-center justify-content-center">
+            <aside>
+                <div class="inner">
+                    <div class="formTitle">
+                        <h2>Edit</h2>
+                    </div>
+                    <hr class="sep">
+                    <form asp-action="Edit" enctype="multipart/form-data">
+                        <div asp-validation-summary="ModelOnly" class="text-danger"></div>
+                        <input type="hidden" asp-for="Id" />
+                        <div class="form-group">
+                            <label asp-for="Title" class="control-label"></label>
+                            <input asp-for="Title" class="form-control" />
+                            <span asp-validation-for="Title" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="Body" class="control-label"></label>
+                            <textarea asp-for="Body" class="form-control"></textarea>
+                            <span asp-validation-for="Body" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="CategoryId" class="control-label"></label>
+                            <select asp-for="CategoryId" class="form-control" asp-items="ViewBag.CategoryId"></select>
+                            <span asp-validation-for="CategoryId" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="CreationTimestamp" class="control-label"></label>
+                            <input asp-for="CreationTimestamp" class="form-control" readonly />
+                            <span asp-validation-for="CreationTimestamp" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label asp-for="LastUpdatedTimestamp" class="control-label"></label>
+                            <input asp-for="LastUpdatedTimestamp" class="form-control" readonly />
+                            <span asp-validation-for="LastUpdatedTimestamp" class="text-danger"></span>
+                        </div>
+                        <div class="form-group">
+                            <label>Upload Article Image</label>
+                            <input type="file" name="ImageFile" id="ImageFile" class="form-control-file">
```

Comment:
replace with this to accept only images.
input type="file" name="ImageFile" id="ImageFile" class="form-control-file" accept="image/x-png,image/jpg,image/jpeg"